src/Object/Api/Mastodon/FollowRequest.php

```diff
@@ -24,7 +25,7 @@ class FollowRequest extends Account
 	 */
 	public function __construct(BaseURL $baseUrl, int $introduction_id, array $publicContact, array $apcontact = [], array $userContact = [])
 	{
-		parent::__construct($baseUrl, $publicContact, $apcontact, $userContact);
+		parent::__construct($baseUrl, $publicContact, new Fields(), $apcontact, $userContact);
```

Comment:
Really "new Fields()" ? Why? I'm just asking :-)

Reply:
Because the constructor takes a Fields collection now but I wasn't interested in adding extra code to verify the contact is a local Friendica user and retrieve their eventual public custom profile fields?